### ISSUE TITLE
physical interaction curation

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#2552]
I noticed physical interactions in Canto look like this:

<img width="645" alt="Screenshot 2022-03-30 at 18 45 21" src="https://user-images.githubusercontent.com/7359272/160906723-d7fcfef2-31a9-4086-9208-aa15896093fc.png">

but  physical interactions should still be the old method, so the genes should be visible and editable

**Comment by kimrutherford** [comment]
Yep, sorry.  It's been that way since we switched over to the new genetic interaction code a year or so ago.  I've been meaning to fix it.

> but physical interactions should still be the old method, so the genes should be visible and editable

When I switched Canto to use new code for handle genetic interactions (which when haven't used since we have a new plan) the physical interactions from Chado broke.  In the new code I changed physical interactions to use genotypes because I didn't understand what I was doing, then I broke a bunch of stuff when I changed things back.  I missed fixing this problem.
